src/components/prompt/prompt.tsx

```diff
@@ -2,21 +2,32 @@ import { Stack, Typography } from "@mui/material";
 import { type FC } from "react";
 
 import { styles } from "./styles.js";
+import { text } from "~/assets/text/index.js";
 
 interface Props {
-  name: string;
-  message: string;
+  stage: string;
+  name: string | undefined;
+  message: string | undefined;
 }
 
-export const Prompt: FC<Props> = ({ name, message }) => {
+export const Prompt: FC<Props> = ({ stage, name, message }) => {
```

Comment:
Component name is a bit generic. It has more logic than just showing what I pass as a message so it would be nice for the name to reflect that